packages/db-authorization/index.js

```diff
@@ -99,6 +109,31 @@ async function auth (app, opts) {
       // fields are specified
       checkSaveMandatoryFieldsInRules(type, rules)
 
+      // We have subscriptions!
+      let userPropToFillForPublish
+      /* istanbul ignore else */
+      if (app.platformatic.mq) {
+        for (const rule of rules) {
+          const checks = rule.find?.checks
+          if (typeof checks !== 'object') {
+            continue
+          }
+          for (const key of Object.keys(checks)) {
+            /* istanbul ignore next */
+            const val = checks[key] || checks[key].eq
```

Comment:
So, we restrict visibility based on `find` rule. I think we need to write this in authorization reference. Also, this works only for `equaliy` checks. I don't think this will work for other cases (like if we resstrict the find to return records from "yesterday", see: https://github.com/platformatic/platformatic/blob/main/packages/db-authorization/test/code-permissions.test.js#L431).
I am fine also in supporting just equality, but if so, we should write this in doc.

Reply:
I'll add it to the docs and add a couple of tests to make this a bit more robust, plus:
1. throw if there is another check other than eq
2. option to skip adding the topics wrappers.